chapter_appendix-mathematics-for-deep-learning/integral-calculus.md

```diff
@@ -341,7 +341,7 @@ $$
 \int_{[a, b]\times[c, d]} f(x, y)\;dx\;dy.
 $$
 
-Suppose that we wish to compute this integral.  My claim is that we can do this by iteratively computing first the integral in say $x$ and then shifting to the integral in $y$, that is to say
+Suppose that we wish to compute this integral.  My claim is that we can do this by iteratively computing first the integral in saying $x$ and then shifting to the integral in $y$, that is to say
```

Comment:
Same as before, removing the word "say" is a better way to correct this writing tic of mine ;).

Reply:
Ok!